### DISCUSSION TITLE
How to rejoin the old pool?

### DISCUSSION THREAD

**Discussion by Mareknown** [Chia-Network/chia-blockchain#16912] [Q&A]
After the last update to v2 my chia did not want to start. I have removed the main folder and synchronized again (it took a few days). Now it is starting but I lost my connection to the pool. I wanted to join it again but it looks like I have to replot all the plots again!? Is there any way to show my existing plots to the old pool?  Am I farming now if the plots where connected to the external pool before? Please help.

**Comment by BrandtH22** [comment]
Hey @Mareknown , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

As for the pool information, it sounds like the wallet will likely need to be resynced for the pool information to properly populate.

From the gui you can go to settings / advanced then click resync. This will close the chia client and upon restarting the wallet will resync. Once fully resynced you should again see the pool information.

From the cli it is generally best to stop chia, rename the wallet db in `~\.chia\mainnet\wallet\db\` , then start chia
